pkg/registry/generic/registry/store.go

```diff
@@ -255,10 +303,10 @@ func (e *Store) Update(ctx api.Context, name string, objInfo rest.UpdatedObjectI
 	}
 
 	out := e.NewFunc()
-
+	var obj runtime.Object
```

Comment:
call this `deletedObj` and only set it in the delete case to make it clear what should be using it outside the func